parachain/integration-test/src/lib.rs

```diff
@@ -0,0 +1,417 @@
+#![cfg(test)]
+#![deny(missing_docs, unused_imports)]
+
+use anyhow::anyhow;
+use ismp::{
+	consensus::StateMachineHeight,
+	events::{Event, StateMachineUpdated},
+	host::{StateMachine, StateMachine::Kusama},
+	messaging::{Message, Proof, ResponseMessage},
+	router::{Request::Get, RequestResponse},
+};
+use sc_service::TaskManager;
+use std::{collections::HashMap, sync::Arc, time::Duration};
+use substrate_state_machine::{HashAlgorithm, StateMachineProof, SubstrateStateProof};
+use subxt::{
+	ext::{
+		codec::{Decode, Encode},
+		sp_core::{sr25519::Pair, Pair as PairT},
+	},
+	tx::PairSigner,
+	utils::AccountId32,
+	OnlineClient,
+};
+use subxt_signer::sr25519::dev::{self};
+use subxt_utils::{
+	gargantua::{
+		api,
+		api::{
+			ismp::events::Request,
+			ismp_demo::events::GetResponse,
+			runtime_types::pallet_ismp_demo::pallet::{GetRequest, TransferParams},
+		},
+	},
+	Hyperbridge,
+};
+use tesseract::logging::setup as log_setup;
+use tesseract_config::AnyConfig;
+use tesseract_messaging::relay;
+use tesseract_primitives::{config::RelayerConfig, IsmpProvider};
+use tesseract_substrate::{config::KeccakSubstrateChain, SubstrateClient, SubstrateConfig};
+use transaction_fees::TransactionPayment;
+
+/// This function is used to fetch get request and construct get response and submit to chain
+/// A(source chain)
+async fn messaging_relayer_lite(
```

Comment:
This is all you need
```suggestion
async fn messaging_relayer_lite(chain_a: SubstrateClient<Hyperbridge>,
	chain_b: SubstrateClient<Hyperbridge>)
```

Reply:
The same goes here,
all the clients have already been derived in the create_clients setup. so am just re using. If i do this, it means all the Online Client and Arc<dyn IsmProvider> clients needs to be derived inside again